tools/travis/run-luacheck.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+set -e
+
+LUA_FILES_TO_CHECK=`find lua_examples -iname "*.lua"`
```

Comment:
Added - but it will take more time to fix all the warnings to be able to enable this feature. But now everyone can use `luacheck` and I now in summer have more free time to fix them one by one :smile: